stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -138,15 +139,41 @@ internal class PaymentOptionsActivity : BasePaymentSheetActivity<PaymentOptionRe
                 }
             }
         }
+    }
+
+    private val label: String
```

Comment:
You can use `by lazy` to make this slightly more efficient - it will evaluate the expression once when this property is first accessed and then always return that value for subsequent calls.

Also maybe `primaryButtonLabel`?

```
private val label: String by lazy {
  resources.getString(
     R.string.stripe_paymentsheet_add_button_label
   )
}
```